frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpanded.tsx

```diff
@@ -27,10 +26,7 @@ const Container = styled.div<{ padTop?: boolean }>`
   padding: 18px 32px 160px;
   gap: 20px;
   ${({ padTop }) => (padTop ? "padding-top: 64px;" : "")}
-  width: 600px;
-  ${({ theme }) => theme.breakpoints.down("md")} {
-    width: 550px;
-  }
```

Comment:
The HTML for the learning resource drawer was roughly
```
<div class="MuiDrawer-paper">
  <div class="our content" />
</div>
```
The paper has a max-width of 600px. The content was exactly 600px wide. The content should be narrower according to Figma. (516px). 

Additionally, when the drawer content was too tall to fit, a vertical scrollbar would appear on the `MuiDrawer-paper`, making the paper width = `600px from content + 15ishpx from scrollbar`. So a vertical scrollbar also caused a horizontal scrollbar to appear.